libcodechecker/libhandlers/analyze.py

```diff
@@ -174,6 +174,14 @@ def is_ctu_capable():
                                     "compiler on the system is special, e.g. "
                                     "when doing cross-compilation.")
 
+    analyzer_opts.add_argument('--capture-analysis-output',
+                               action='store_true',
+                               default=False,
```

Comment:
For extension arguments like these, `argparse.SUPPRESS` is used as the default, so `(Default: False)` is not printed to the help.

Be careful, with this change, you need to change `args.dest` to `'dest' in args` everywhere, as a non-specified optstring results in a key error.